_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -501,33 +411,23 @@ This [HTML document][] has a [keyboard shortcut][] using only a [printable chara
 </html>
 ```
 
-### Inapplicable
+#### Inapplicable Example 4
 
-#### Inapplicable Example 1
-
-This document is not an [HTML document][].
-
-```html
-<svg height="100" width="100">
-  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
-  Sorry, your browser does not support inline SVG.  
-</svg>
-```
-
-#### Inapplicable Example 2
-
-This [HTML document][] has a [keyboard shortcut][] that requires pressing one [non-printable character][non-printable characters].
+This [HTML document][] has a [keyboard event][] that is not [dispatched][] to an [event target][].
```

Comment:
I guess the event could be listened only by the `window` object. That would not be in the HTML page and thus be good…